src/Item/Item.router.ts

```diff
@@ -37,6 +37,8 @@ import {
   UnauthorizedToChangeToCollectionError,
   UnauthorizedToUpsertError,
 } from './Item.errors'
+import { ItemCuration } from '../Curation/ItemCuration'
+import { ItemCurationAttributes } from '../Curation/ItemCuration/ItemCuration.types'
```

Comment:
This can be merged into one I think! and also, can we move them a bit higher? (after all `./`s)